code/game/objects/structures/railings.dm

```diff
@@ -74,17 +83,6 @@
 		return . || mover.throwing || mover.movement_type & (FLYING | FLOATING)
 	return TRUE
 
-/obj/structure/railing/corner/CanPass()
-	..()
-	return TRUE
-
-/obj/structure/railing/proc/init_connect_loc_element()
-	var/static/list/loc_connections = list(
```

Comment:
why is this de-procced?

Reply:
we removed the call to it in `/obj/structure/railing/Initialize(mapload)`